<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMediaDispersive</class>
 <widget class="QMainWindow" name="frmMediaDispersive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>384</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>色散材料</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>材料名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMediaName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>59</y>
      <width>376</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>371</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddRow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>80</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>增加频点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoveRow">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>80</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>删除选中行</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
